### PR TITLE
feat: Allows for a user to utilize self-signed certificates

### PR DESCRIPTION
**Pull request by davcpas1234**
# New Feature
# Supports  #90 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Allows the ability for a user to utilise self-signed certificates with SendGrid
- Current Client implementation will fail to process a `curl_init()` function if, for example, the user is testing the implementation on a local server with an unknown CA

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=101) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=101) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/php-http-client?pullRequest=101) it.</sub>

**Comment by thinkingserious** [comment]
Thanks @davcpas1234! This one is our backlog for a code review.

File: lib/Client.php [hunk 1]
```diff
@@ -304,6 +311,20 @@ public function setRetryOnLimit($retry)
...
     }
 
+    /**
+     * Set default verify certificates flag
+     *
+     * @param bool $verifySSLCerts
+     *
+     * @return Client
+     */
+    public function setVerifySSLCerts($verifySSLCerts)
```
Comment (peter279k) [inline lib/Client.php]: It only has the `setVerifySSLCerts` method.

Do we need to add the `getVerifySSLCerts` method?

I think we have this method to get current `$verifySSLCerts` value.

Comment (childish-sambino) [reply lib/Client.php:323]: I think it's fine as-is. We don't have getters for the other options.

**Review by childish-sambino — APPROVED** [review]

**Pull request merged**